nats-server/clientsz.yml

```diff
@@ -18,3 +18,24 @@ observation:
         source: limit
         type: long_gauge
         label: max channels
+
+  - name: clientsz_client_subscription
+    metricNamespace: nats
+    path: $.clients[?(@.id=${clientName}})].subscriptions.${subscriptionName}
+
+    metric:
+      - name: streaming.clients.subscriptions.pending.count
+        source: counter
+        type: long_gauge
+        label: pending count
```

Comment:
Better `pending subscriptions`.

Reply:
this is Messages count of one subscription.

Reply:
Change to pending messages. add also description